### DISCUSSION TITLE
Input Image errors

### DISCUSSION THREAD

**Discussion by pradeep-04** [lllyasviel/Fooocus#2424] [Q&A]
I can't understand what's wrong but I can just only use the image prompt without its advanced settings. once I apply advanced settings and try image swap or something else it takes a little time and shows some error messages and text "error" at the center of the image field. the same happens when I use other things like inpaint or outpatient. Can anyone help with this?

**Comment by mashb1t** [comment]
Please check the terminal output and post the full log from application start to error here or in a new issue.
